husarion_ugv_bringup/launch/bringup.launch.py

```diff
@@ -172,36 +164,12 @@ def generate_launch_description():
         }.items(),
     )
 
-    husarion_ugv_bringup_common_dir = PythonExpression(
-        [
-            "'",
-            common_dir_path,
-            "/husarion_ugv_bringup' if '",
-            common_dir_path,
-            "' else '",
-            FindPackageShare("husarion_ugv_bringup"),
-            "'",
-        ]
-    )
-
-    gamepad_launch = IncludeLaunchDescription(
+    teleop_launch = IncludeLaunchDescription(
         PythonLaunchDescriptionSource(
             PathJoinSubstitution(
-                [FindPackageShare("joy2twist"), "launch", "gamepad_controller.launch.py"]
+                [FindPackageShare("husarion_ugv_teleop"), "launch", "teleop.launch.py"]
             )
-        ),
-        launch_arguments={
-            "log_level": log_level,
-            "namespace": namespace,
-            "joy2twist_params_file": PathJoinSubstitution(
-                [
-                    husarion_ugv_bringup_common_dir,
-                    "config",
-                    PythonExpression(["'joy2twist_", robot_model_name, ".yaml'"]),
-                ]
-            ),
-        }.items(),
-        condition=IfCondition(launch_gamepad),
+        )
```

Comment:
```suggestion
          launch_arguments={"launch_gamepad": "True"}.items(),
        )
```

For hardware configuration the gamepad should always start up.